pipelines/_ingredients/cpPtcLSST.yaml

```diff
@@ -0,0 +1,59 @@
+description: cp_pipe ptc calibration construction.
+tasks:
+  cpPtcIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "cpPtcIsrExp"
+      expectWcs: false
+      doDefect: true
+      doBias: false
+      doDark: false
+      doFlat: false
+      doDiffNonLinearCorrection: false
+      doBootstrap: true
+      doDeferredCharge: false
+      doBrighterFatter: false
+      doLinearize: true
```

Comment:
I think I'm confused as to the ordering of PTC and linearity.  Linearity (from the previous pipeline) is derived from a PTC generated without linearity and without applyGains.  We then re-derive a linearity corrected PTC here, and use that to determine the actual PTC values that we'll use for gains/read noise/saturation.  I assume we'll then have a `cp_verify` pipeline that rederives a PTC from the linearity and gain-corrected exposures, and use that to determine the stability of our values (as that verify PTC should have gains of 1.0 if nothing has drifted).

Reply:
After we have a linearizer we can apply it (in ADU at this point) to the next round of "final" PTC fitting.  And this is indeed what we use to determine the actual PTC values to use for gains/read noise/saturation (TBD).
That's a good question about cp_verify ... there is no additional pipeline of that sort right now but there should be.